IntroToCSharp/Pages/Activities/SettingBuilder.razor

```diff
@@ -0,0 +1,150 @@
+    </Text>
+    <Text>
+        Below is a list of different city sizes that might help the user decided if their place is a city, town or something different.
+    </Text>
+
+    <CitySizes/>
+
+    <TaskItem Name="Ask for city or town size"><Code>Create a prompt to the user for the city or town size.</Code></TaskItem>
+    <TaskItem Name="Display Location Sizes"><Code>Display a list of possible sizes to help user pick the size of their location.</Code></TaskItem>
+    <TaskItem Name="Store location size"><Code>Store the users input as "locationSize".</Code></TaskItem>
+
+    <Text>
+        It might also be helpful at this point to capture just how many people are in this town or area! The population of a particular
+        location in a game might increase or decrease over the course of the game but for now we are just going to get a starting population.
+    </Text>
+
+    <TaskItem Name="Ask for population"><Code>Create a prompt to the user for the location population.</Code></TaskItem>
+    <TaskItem Name="Store location population"><Code>Store the location population as an <b>integer</b></Code></TaskItem>
+
+    <Text>
+        We are well on our way to make our first location in a game! What we need now is to build the area or "geography" around our
+        location. When we think of the location what kind of natural reasources surround it? Is it a small village in the center of a forest?
+        A colony in space surrounded by asteroid fields? An underground civilization built within a mine filled with enormous crystals?
+    </Text>
+    <Text>
+        We also might want to know what natural resources surround the location! Is is close to the ocean? Are there a lot of animals in the area
+        for hunting? Perhaps it is in a metropolis surrounded by nothing but desert?
+    </Text>
+
+    <TaskItem Name="Ask for geography"><Code>Create a prompt to the user for the geography.</Code></TaskItem>
+    <TaskItem Name="Store geography"><Code>Store the geography of the location as "geography".</Code></TaskItem>
+    <TaskItem Name="Ask for geography"><Code>Create a prompt to the user for the natural resources around the location.</Code></TaskItem>
+    <TaskItem Name="Store natural resources"><Code>Store the natural resources as "naturalResources".</Code></TaskItem>
+
+    <Section Title="Displaying the information">
+    <Text>
+        Wow! We've got a lot of information going now! I think it is about time to start looking at how to Display the information
+        in a way that looks good for our user. While we could just Display the information out as "information: ", "user input" format,
+        it might be a little hard to read the information. Especially if we want to add more information.
+    </Text>
+    <Text>
+        What we should do instead is change the color of the information in each row to make it a little easier to read!
```

Comment:
Maybe instead of "we should", maybe something like: "You can make your program easier to read by highlighting the information with colors."